ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -36,6 +36,17 @@ import {
   TextVariant,
 } from '../../../helpers/constants/design-system';
 
+// Not all browsers have usb support. In particular, Firefox does
+// not support usb. More information on that can be found here:
+// https://mozilla.github.io/standards-positions/#webusb
+//
+// The below `&& window.navigator.usb` condition ensures that we
+// only attempt to connect Trezor via usb if we are in a browser
+// that supports usb. If not, the connection of the hardware wallet
+// to the browser will be handled by the Trezor connect screen. In
+// the case of Firefox, this will depend on the Trezor bridge software
+const isUSBSupported = !process.env.IN_TEST && window.navigator.usb;
```

Comment:
without this additional `!process.env.IN_TEST` we can't bypass the USB permission dialog prompted by the browser (or at least I wasn't able to find a viable alternative)

Reply:
nit: maybe we can add this as a comment?